capa/rules/__init__.py

```diff
@@ -720,11 +768,12 @@ def _extract_subscope_rules_rec(self, statement):
                 name = self.name + "/" + uuid.uuid4().hex
                 new_rule = Rule(
                     name,
-                    subscope.scope,
+                    Scopes(subscope.scope, DEV_SCOPE),
                     subscope.child,
                     {
                         "name": name,
-                        "scope": subscope.scope,
+                        "scopes": Scopes(subscope.scope, DEV_SCOPE).__dict__,
```

Comment:
```suggestion
                        "scopes": dataclasses.asdict(Scopes(subscope.scope, DEV_SCOPE)),
```

Reply:
generally prefer to avoid calling methods that begin with `_` as they're intended to be private.